src/activities/quizzes/QuizEntity.js

```diff
@@ -320,6 +364,7 @@ export class QuizEntity extends Entity {
 	equals(quiz) {
 		const diffs = [
 			[this.name(), quiz.name],
+			[this.isShuffleEnabled(), quiz.shuffle],
```

Comment:
Are you going to wait til the activities PR has been added before merging this one? Otherwise `quiz.shuffle` will be `undefined `and will always trigger dirty check

Reply:
I'll put up the `activities` PR in a bit. Thanks for the reminder!

Reply:
Wanted to use your refactored checkbox css styles